SU-SWS/decanter-web
# D6: Upgrading all the things (D6.3.0)

# READY FOR REVIEW

# Summary
- Updates to the latest and greatest
- Next 13
- There are several things I am not proud of in here but the idea was to get it to build with updated packages to give it some life for a while. 

# Needed By (Date)
- September 15th

# Urgency
- Minor

# Steps to Test

1. Review code
2. Navigate to the [preview build](https://deploy-preview-142--elegant-poitras-87214a.netlify.app/)
3. Do some linky clicky
4. Ensure the components work
5. Run some side/side comparisons to the live site

Feel free to call out any particularly heinous choices here. I definitely cut all the corners and skipped best practices in favor of just getting it done.
